src/ant/include/ant/AntennaChecker.hh

```diff
@@ -122,6 +118,13 @@ struct Violation
   int diode_count_per_gate;
 };
 
+using LayerInfoMap = std::map<odb::dbTechLayer*, NodeInfo>;
+using GraphNodeVector = std::vector<GraphNode*>;
+using GraphNodeVectorMap
+    = std::unordered_map<odb::dbTechLayer*, GraphNodeVector>;
+using GateInfoMap = std::map<std::string, LayerInfoMap>;
+using ViolationList = std::vector<Violation>;
```

Comment:
GraphNodeVector is a vector and so is ViolationList.  I find that confusing.  We don't have a standard naming scheme here (and I don't see one in the Google style guide).  

My personal preference is to using a plural for simple collections, eg GraphNodes and "To" for mapping, eg LayerToNodeInfo.  You can combine it for LayerToGraphNodes (replacing GraphNodeVectorMap).  That way you don't have to rework the code if you change a vector to a deque or a map to an unordered_map, etc.